lib/seed/runner.rb

```diff
@@ -35,6 +35,8 @@ def call
       result = FacilitySeeder.call(config: config)
       total_counts[:facility] = result&.ids&.size || 0
       UserSeeder.call(config: config)
+      announce "Seeding drug stocks..."
+      seed_drug_stocks
```

Comment:
The `FacilitySeeder` sometimes does not create new facilities, if enough facilities exist already. I think it should be ok for us to create extra drug stock records, since we only surface "most recent per month" anyway. Just wanted to flag here to be sure.

Reply:
Yup, this should be fine, we'll just have some extra drug stock records for the month when its run.